src/python/gudhi/representations/vector_methods.py

```diff
@@ -510,24 +510,27 @@ def transform(self, X):
         for i in range(num_diag):
             orig_diagram, diagram, num_pts_in_diag = X[i], new_X[i], X[i].shape[0]
             try:
-                new_diagram = DiagramScaler(use=True, scalers=[([1], MaxAbsScaler())]).fit_transform([diagram])[0]
+                #new_diagram = DiagramScaler(use=True, scalers=[([1], MaxAbsScaler())]).fit_transform([diagram])[0]
+                new_diagram = DiagramScaler().fit_transform([diagram])[0]
```

Comment:
Does DiagramScaler do anything with no scalers?

Reply:
I was not sure of this part, let's see what Mathieu thinks

Reply:
Sorry for the very late reply. Yes, this line is useless now, you can remove it.